dashboard/app/views/levels/editors/_grid.html.haml

```diff
@@ -1,3 +1,6 @@
+- content_for(:head) do
+  %script{src: minifiable_asset_path('js/levels/editors/_grid.js')}
```

Comment:
This isn't ideal, because we're still relying on GridEditor to set a global variable, and then we're accessing that global variable from inline js later in this haml file, but it does move us in the right direction.